### ISSUE TITLE
503 error: The Lambda function associated with the CloudFront distribution is invalid

### ISSUE THREAD

**Issue by MelMacaluso** [serverless-nextjs/serverless-next.js#584]
**Describe the bug**
After logging in with the OAUTH google's panel (using [next-auth](https://github.com/nextauthjs/next-auth)) when redirected back to the site I am presented with a 503 error. Everything works fine locally. Presuming is some misconfig in cloudfront / lambda / labmda@edge. 

Here's the full error: 

![image](https://user-images.githubusercontent.com/24974216/92235639-507b4000-eeb4-11ea-9725-3b5b6179b4b0.png)

**To Reproduce**

1. Attempt to login through out of the box OAUTH google provider from [next-auth](https://github.com/nextauthjs/next-auth)
2. Enter your credentials on google's login panel
3. Be redirected back to the site
4. See 503 error

**Additional context**

Package.json

```json
{
  "dependencies": {
    "next": "latest",
    "next-auth": "^3.1.0",
  }
}
```

Serverless.yml

```yaml
appName:
  component: '@sls-next/serverless-component@1.17.0-alpha.6'
  inputs:
    name: appName
    bucketName: app-name-api-bucket
    runtime:
      defaultLambda: nodejs12.x
      apiLambda: nodejs12.x
    bucketRegion: eu-west-2
```
**Cloudwatch logs are not showing any errors.**

**Comment by MelMacaluso** [comment]
To anyone seeing this, this error was actually caused by an issue I had with next-auth.js where I specified a database but my inbound rules were too strict to let the lambda write to it.

**Comment by cedricmckinnie** [comment]
> To anyone seeing this, this error was actually caused by an issue I had with next-auth.js where I specified a database but my inbound rules were too strict to let the lambda write to it.

To add to this: I also ran into this problem too but for me it was due to RDS Aurora Serverless. 

The problem for me was that the database Cold Start time took about one second longer than the 30 sec lambda@edge timeout when redirecting back to the page from Google OAuth provider.

The solution for me was to disable AutoPause of Serverless database to keep the database from cold starting which was fine for my use case because it's a production environment anyway.
